pipeline/data/importers/corpus/bucket.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""
+Import data from a Google Cloud storage bucket.
```

Comment:
I see your idea of the bucket importer but since we use a public bucket you can easily do 
```
wget https://storage.googleapis.com/releng-translations-dev/data/en-ca/catalan_textual_corpus.ca.zst
```
so you don't really need a GCP client here. Instead you could add support for downloading from a regular http url with `requests`, then it would be more flexible where to download from. Also getting rid of an extra pypi dependency can help in to prevent resolution conflicts in future.